app/models/membership.rb

```diff
@@ -1,5 +1,6 @@
 class Membership < ApplicationRecord
   belongs_to :supplier
+  belongs_to :user, optional: true
```

Comment:
should this be `optional`?

Reply:
Unfortunately, it does for a short while, while we don't have the users imported yet. It's a bit of a chicken and egg scenario. Without the users imported, all the membership models will be invalid without the associated User model existing.

My proposed solution here is to make this optional, and I have locally run the specs without my spec changes in the PR, which pass. Alternatives are to not create this `belongs_to` until after we import the users, or additionally, not update the specs until we add this belongs_to is added. Or do what I've suggested here.

Reply:
Right, good point, without it we'd have problems adding new users in the meantime... So it would be a case of removing the optional bit once we have the user data imported? I'm happy with that.

Reply:
Yes, that's the plan.